### PR TITLE
try again of #3003 (curve fi fixes and add bitcoin cex)

### PR DESCRIPTION
**Pull request by andrewhong5297**
Brief comments on the purpose of your changes: Not sure why CI for #3003 kept failing, so doing a new branch to try again.

**For Dune Engine V2**

I've checked that:

### General checks:
* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [x] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [x] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each sql file is a select statement and has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`
* [x] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [x] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [x] `coin_id` represents the ID of the coin on coinpaprika.com
* [x] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [x] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [x] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [x] where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Comment by andrewhong5297** [comment]
okay creating a new branch worked!

**Comment by jeff-dude** [comment]
> okay creating a new branch worked!

not even going to speculate on that other branch, not worth it 🥲

File: models/prices/solana/prices_solana_tokens.sql [hunk 1]
```diff
@@ -15,7 +15,7 @@ SELECT
 FROM
 (
     VALUES
-
+    ("hades","solana","HADES","BWXrrYFhT7bMHmNBFoQFWdsSgA3yXoAnMhDK6Fn1eSEn",9),
```
Comment (jeff-dude) [inline models/prices/solana/prices_solana_tokens.sql:18]: this is not a valid id value on coinpaprika -- can you find the correct one? if you search on their site, usually the url will contain the id. i saw `hades-hades` but that is on ethereum and not active

**Comment by andrewhong5297** [comment]
wait pause on this one I need to double check the solana prices tokens stuff

File: models/prices/solana/prices_solana_tokens.sql (hunk 1 above)
Comment (andrewhong5297) [reply models/prices/solana/prices_solana_tokens.sql:18]: ah yeah was just checking

Comment (andrewhong5297) [reply models/prices/solana/prices_solana_tokens.sql]: okay I've added only tokens that are active on coinpaprika now!

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**